.github/workflows/build-variant.yaml

```
@@ -57,7 +57,7 @@ jobs:
       updates: ${{ steps.build.outputs.updates }}
       digest: ${{ steps.image.outputs.digest }}
     container:
-      image: ghcr.io/eeems/arkes-builder:${{ inputs.builder || github.head_ref || github.ref_name }}
+      image: ghcr.io/eeems/arkes-builder:${{ inputs.builder || replace(replace(replace(replace(replace(replace(github.head_ref || github.ref_name, ' ', '-'),'/', '-'),'@', '-'),':', '-'),'^[.]+', ''),'^-]+', '')}}
```

actionlint (1.7.11)

[error] undefined function "replace". available functions are "always", "cancelled", "case", "contains", "endswith", "failure", "format", "fromjson", "hashfiles", "join", "startswith", "success", "tojson"

(expression)